gravitee-apim-console-webui/src/management/application/creation-ng/application-creation.component.ts

```diff
@@ -58,6 +59,13 @@ const TYPES_INFOS = {
   },
 };
 
+function toMetadataDictionary(additionalClientMetadata: Header[]) {
+  return additionalClientMetadata.reduce((acc, header) => {
+    acc[header.key] = header.value;
```

Comment:
it means it will take the last value in case of duplicate key somewhere. it should probably be written somewhere